spec/lib/percy/capybara/loaders/base_loader_spec.rb

```diff
@@ -14,6 +14,18 @@
       expect(resource.content).to include('Hello World!')
       expect(resource.sha).to eq(Digest::SHA256.hexdigest(resource.content))
     end
+    it 'includes an iframe resource' do
```

Comment:
Looks like we'll need `js: true` indeed, I get this with the Rack app:
`Capybara::NotSupportedByDriverError: Capybara::Driver::Base#within_frame`